src/utils/ContractUtils.ts

```diff
@@ -16,6 +16,15 @@ export function setContractDefaults(contract: Contract, config: Config): Contrac
   return contract
 }
 
+export async function networkSupportsEip1559(web3: Web3): Promise<boolean> {
+  const feeHistory = await web3.eth.getFeeHistory(4, 'pending', [25, 50, 75])
+  if (feeHistory && feeHistory.baseFeePerGas) {
```

Comment:
way shorter and only one return:

```js
return feeHistory?.baseFeePerGas ? true : false
```

Reply:
aa forgot to delete this sorry my bad, the logic is integrated in the `sendTx` indeed